src/index.ts

```diff
@@ -97,13 +97,13 @@ async function main(): Promise<void> {
     ]);
 
     const httpApi = createHttpApi({
-      storageDir: config.storageDir,
+      storageDir: config.chainDataDir,
```

Comment:
is this confusing? the storage dir inside the http api context is the chain data directory, but it could be confused with the root storage dir because of the name

Reply:
Maybe it could be:

```ts
const httpApi = createHttpApi({
  httpRootDir: config.chainDataDir
```

So as to keep "storage dir" to one meaning?

Reply:
updated :v: